source/github-helpers/search-query.ts

```diff
@@ -24,6 +24,8 @@ function cleanQueryParts(parts: string[]): string[] {
 	return deduplicateKeywords(parts, 'is:issue', 'is:pr');
 }
 
+const labelLinkRegex = /\/labels\/([^/]+)$/;
```

Comment:
Still matches /fregante/labels/issues. It should also include the the beginning, like `^[/][^/]+[/][^/]+[/]labels[/][^/]+[/]?$`